### DISCUSSION TITLE
Bug report: downlaoding Energy Dashboard "day data" is one day off

### DISCUSSION THREAD

**Discussion by eagle257** [home-assistant/frontend#21064] [Configuration] [answered]
Hi, I wanted to report a bug

When downloading the day-data from my Energy Dashboard (i.e. selecting 23.5 - 31.05), the exported data in excel is misplaced by one day. I.e. the data that is correctly shown within HomeAssistant Energy Dashboard for May 23 is shown in Excel as May 22. The Excel output data range shows 22.05 - 30.05 instead of 23.05 - 31.05.

**Comment by karwosts** [comment, marked as answer]
Did you look at the full timestamp? The log times are in UTC, so the day start will give a specific moment in UTC, which may appear to be the previous day if you only look at the date and ignore the hour/TZ. 

In general bugs should be reported to issues (not discussions), but my guess is this is not a bug but just a misunderstanding.
